package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@playcanvas/pcui",
-      "version": "2.3.1",
+      "version": "2.3.3",
```

Comment:
I don't think this matters but doing npm install will probably bump this to 2.3.4 to match the package.json.